app/controllers/users_controller.rb

```diff
@@ -1,3 +1,29 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = t("controllers.users.welcome_sample_app")
+      redirect_to @user
+    else
+      render 'new'
+    end
+  end
+
+  private
+
+    def user_params
```

Comment:
our style:
- No blank line after `private` (optional)
- functions have same alignment with `private`
Please try `rubycop` if you have time:
https://github.com/rubocop-hq/rubocop